plugins/modules/source_control/gitlab/gitlab_group_variable.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Florent Madiot (scodeman@scode.io)
+# Based on code:
+# Copyright: (c) 2019, Markus Bergholz (markuman@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_group_variable
+short_description: Creates/updates/deletes GitLab Groups Variables
+version_added: 1.2.0
+description:
+  - When a group variable does not exist, it will be created.
+  - When a group variable does exist, its value will be updated when the values are different.
+  - Variables which are untouched in the playbook, but are not untouched in the GitLab group,
+    they stay untouched (I(purge) is C(false)) or will be deleted (I(purge) is C(true)).
+author:
+  - "Florent Madiot (@scodeman)"
+requirements:
+  - python >= 2.7
+  - python-gitlab python module
+extends_documentation_fragment:
+  - community.general.auth_basic
+
+options:
+  state:
+    description:
+      - Create or delete group variable.
+      - Possible values are present and absent.
+    default: present
+    type: str
+    choices: ["present", "absent"]
+  api_token:
+    description:
+      - GitLab access token with API permissions.
+    required: true
+    type: str
+  group:
+    description:
+      - The path and name of the group.
+    required: true
+    type: str
+  purge:
+    description:
+      - When set to true, all variables which are not untouched in the task will be deleted.
```

Comment:
`not untouched` == `touched`? Or do you mean `not mentioned` in the `vars` option?

Reply:
Same as above,  same paragraph as for the current [gitlab_project_variable.py](https://github.com/ansible-collections/community.general/blob/main/plugins/modules/source_control/gitlab/gitlab_project_variable.py)

this is saying that variables created/modified manually in gitlab but not mentionned in the ansible task will be removed from gitlab if purge is set to True

Reply:
I guess both sentences should be rewritten in both modues then :) Fine for me if this is done in a later PR though.